### PR TITLE
Add ECS fields to process & FS resources

### PR DESCRIPTION
**Pull request by uri-weisman**
The work has been done as part of the ECS mapping epic:

- https://github.com/elastic/security-team/issues/3788
- https://github.com/elastic/security-team/issues/3787

### CODE REVIEW

File: resources/fetchers/process_fetcher.go [hunk 1]
```diff
@@ -40,15 +43,65 @@ const (
 	CMDArgumentMatcher  = "\\b%s[\\s=]\\/?(\\S+)"
 	ProcessResourceType = "process"
 	ProcessSubType      = "process"
+	userHz              = 100
```
Comment (jeniawhite) [inline resources/fetchers/process_fetcher.go:46]: The value of HZ varies across kernel versions and hardware platforms.
There is an assumption here for certain conditions that might not be respected between different deployments and versions.
I think that we can use https://man7.org/linux/man-pages/man3/sysconf.3.html and pull the value of `_SC_CLK_TCK` in order to reduce this assumption.

Comment (uri-weisman) [reply resources/fetchers/process_fetcher.go:46]: Thank you for raising this up @jeniawhite .
It is a borrowed implementation from [go-sysinfo](https://github.com/elastic/go-sysinfo), we decided to deliver it with this known limitation while opening 2 new tickets to the go-sysinfo repo.
- https://github.com/elastic/go-sysinfo/issues/122
- https://github.com/elastic/go-sysinfo/issues/12

Once those tasks will be delivered, possibly by us, we'll change our implementation to use the go-sysinfo pkg.

**Review by jeniawhite — APPROVED** [review]

**Review by DaveSys911 — APPROVED** [review]
LGTM

File: .github/workflows/unit-test.yml [hunk 2]
```diff
@@ -78,6 +78,9 @@ jobs:
     runs-on: ubuntu-latest
     timeout-minutes: 15
     steps:
+      - uses: actions/setup-go@v3
+        with:
+          go-version: 1.18
```
Comment (DaveSys911) [inline .github/workflows/unit-test.yml:83]: Could you please set go version from using the env var like the unit tests job?

**Review by DaveSys911 — CHANGES_REQUESTED** [review]

File: resources/fetchers/kube_provider.go [hunk 3]
```diff
@@ -80,6 +80,8 @@ func (r K8sResource) GetMetadata() fetching.ResourceMetadata {
 	}
 }
 
+func (r K8sResource) GetElasticCommonData() any { return nil }
```
Comment (oren-zohar) [inline resources/fetchers/kube_provider.go:83]: shouldn't this sit in the kube_fetcher?

Comment (uri-weisman) [reply resources/fetchers/kube_provider.go:83]: Seems fine to me to reside next to the rest of the `Resource interface`
```
type Resource interface {
	GetMetadata() ResourceMetadata
	GetData() any
	GetElasticCommonData() any
}
```

If you feel this is not the place we can move them all.

File: .github/workflows/unit-test.yml (hunk 2 above)
Comment (uri-weisman) [reply .github/workflows/unit-test.yml]: Fixed

**Review by DaveSys911 — APPROVED** [review]

**Pull request merged**